ccf_transaction_processor/transaction_processor/enclave_registry.h

```diff
@@ -21,6 +21,52 @@ using namespace std;
 namespace ccf
 {
 
+  struct ContractEnclaveAttestationVerificationPolicy {
+    bool check_attestation;
+    string mrenclave;
+    string basename;
+    string ias_public_key;
+  };
+
+  DECLARE_JSON_TYPE(ContractEnclaveAttestationVerificationPolicy);
+  DECLARE_JSON_REQUIRED_FIELDS(ContractEnclaveAttestationVerificationPolicy,
+    check_attestation,
+    mrenclave,
+    basename,
+    ias_public_key);
+
+
+  struct ProofData{
+    vector<string> certificates;
+    string verification_report;
+    string signature;
+  };
+  DECLARE_JSON_TYPE(ProofData);
+  DECLARE_JSON_REQUIRED_FIELDS(ProofData,
+  certificates,
```

Comment:
style request: make all the parameters line up including ProofData if you are going to put them on separate lines.

Reply:
thank you, done